frontend/src/components/ShootDetails/GShootInfrastructureCard.vue

```diff
@@ -346,6 +353,8 @@ export default {
       getResourceRefName,
       credential,
       isSharedBinding,
+      availableFloatingPools,
+      openStackDomainName,
```

Comment:
```suggestion
```

Sorry my fault :) We don't need this int he component, as `availableFloatingPools` which requires `` is already defined inside setup function.